website/docs/reference/sinks/loki.md.erb

```diff
@@ -0,0 +1,34 @@
+<%- component = metadata.sinks.loki -%>
+
+<%= component_header(component) %>
+
+## Configuration
+
+<%= component_config_example(component) %>
+
+## Options
+
+<%= fields(component.specific_options_list, heading_depth: 3) %>
+
+<%- if component.env_vars_list.any? -%>
+## Env Vars
+
+<%= fields(component.env_vars_list) %>
+
+<%- end -%>
+## How It Works [[sort]]
+
+### Decentralized deployments
+
+Loki currently does not support out-of-order inserts.  If Vector is deployed in a decentralized setup
```

Comment:
Good call !

Reply:
FYI: we're trying to improve this. but it's hard cause it's the foundation of the db behind. One option we're looking into is being able to let loki rework/create timestamp server side.

Reply:
A bit off topic, but I am a huge fan of how facebooks logdevice handles multiple sequencers (ingestors) by giving up on a global ordering of timestamps but using a combination of (seq_id, ts) which would help with this problem. 

https://logdevice.io/docs/Writepath.html#sequencer

Excited to see what you all come up with! :) For now I think the best is to just run things through a central vector instance, there might be some funky hacky solutions around this but I think centralized is good enough for now.